Formula/odin.rb

```diff
@@ -1,17 +1,13 @@
 class Odin < Formula
   desc "Programming language with focus on simplicity, performance and modern systems"
   homepage "https://odin-lang.org/"
-  url "https://github.com/odin-lang/Odin/archive/v0.13.0.tar.gz"
-  sha256 "ae88c4dcbb8fdf37f51abc701d94fb4b2a8270f65be71063e0f85a321d54cdf0"
-  license "BSD-2-Clause"
-  revision 1
+  url "https://github.com/odin-lang/Odin.git",
+      tag:      "dev-2022-05",
+      revision: "df233aee942bd85a5162a36a82bf33fe74d2f2ad"
+  version "2022-05"
```

Comment:
so the version scheme got changed to CalVer?

Reply:
At least during the beta (pre-1.0) phase. Upstream plans to switch to semver again once 1.0 is released (https://github.com/odin-lang/Odin/issues/887).

Reply:
I have some concern that we will run into `version_scheme` issue down the path.